src/controllers/organizationController.ts

```diff
@@ -87,7 +88,10 @@ export default class OrganizationController {
     | IResponseSuccessRedirectToResource<Organization, Organization>
   > {
     return withUserFromRequest(req, async user => {
-      if (user.role !== UserRoleEnum.ORG_DELEGATE) {
+      const userPermission = accessControl
```

Comment:
```suggestion
      const createOwnOrganization = accessControl
```

Reply:
or, better, extract and reuse:

`const userPerm = accessControl.can(user.role);`